integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java

```diff
@@ -44,19 +44,37 @@ void testCreateAndLoadSchema() {
     testDatabaseName = "t_create2";
     sql(
         String.format(
-            "CREATE DATABASE %s COMMENT 'comment' LOCATION '/user'\n"
+            "CREATE DATABASE %s COMMENT 'comment' LOCATION '/tmp/test_%s'\n"
```

Comment:
It would be better to use a common variable `databaseLocation` to create database and check location